doc/develop/getting_started/index.rst

```diff
@@ -533,6 +533,12 @@ that are used to emulate, flash and debug Zephyr applications.
             sudo cp ~/zephyr-sdk-0.14.2/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
             sudo udevadm control --reload
 
+      #. Make sure the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` and :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables
+         are set correctly (to ``zephyr`` and your Zephyr SDK installation directory, respectively).
+         See the :ref:`toolchain_zephyr_sdk_bundle_variables` section for details.
```

Comment:
See https://github.com/zephyrproject-rtos/zephyr/pull/46319#discussion_r890599871 and https://github.com/zephyrproject-rtos/zephyr/pull/46319#discussion_r890602240